src/main/webapp/app/exercises/shared/result/result-detail.component.ts

```diff
@@ -475,13 +490,19 @@ export class ResultDetailComponent implements OnInit {
                 return feedbackItem.type === FeedbackItemType.Test && feedbackItem.positive && !feedbackItem.text;
             });
             if (positiveTestCasesWithoutDetailText.length > 0) {
+                this.numberOfAggregatedTestCases = positiveTestCasesWithoutDetailText.length;
                 return [
                     {
                         type: FeedbackItemType.Test,
-                        category: 'Feedback',
-                        title: positiveTestCasesWithoutDetailText.length + ' passed test' + (positiveTestCasesWithoutDetailText.length > 1 ? 's' : ''),
+                        category: this.showTestDetails
+                            ? this.translateService.instant('artemisApp.result.detail.test.name')
+                            : this.translateService.instant('artemisApp.result.detail.feedback'),
```

Comment:
Looking at those translations - would it make more sense to move this to the template? The component would need to be restructured then, though.

It's way out of scope for this PR, but I'd like to hear your opinion on that.

Reply:
I agree, a follow up might want to improve on this